crates/ruff_server/src/session/options.rs

```diff
@@ -283,11 +287,13 @@ impl Combine for LintOptions {
 #[serde(rename_all = "camelCase")]
 struct FormatOptions {
     preview: Option<bool>,
+    backend: Option<FormatBackend>,
 }
```

Comment:
Including it directly in the `format` table might indicate that this is a stable option, I think there are two ways to communicate the experimental nature of this feature:
1. Make it explicit in the documentation
2. Include it under `experimental.format.backend` similar to ty's `experimental` namespace (https://docs.astral.sh/ty/reference/editor-settings/#experimental)